pkg/volume/awsebs/attacher_windows.go

```diff
@@ -0,0 +1,67 @@
+// +build windows
+
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package awsebs
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+)
+
+func (attacher *awsElasticBlockStoreAttacher) getDevicePath(volumeID, partition, devicePath string) (string, error) {
+	return attacher.getDiskNumber(volumeID)
+}
+
+// getDiskNumber gets the Windows disk number for a given volume ID. The disk number is needed for mounting.
+// TODO This only works for Nitro-based instances
+// TODO fallback to Get-Disk
+func (attacher *awsElasticBlockStoreAttacher) getDiskNumber(volumeID string) (string, error) {
+	// Split the ID from zone: aws://us-west-2b/vol-06d0909eb358b05f9
+	split := strings.Split(volumeID, "/")
+	volumeID = split[len(split)-1]
+
+	// ebsnvme-id is present on AWS-provided Windows Server AMIs
+	// https://docs.aws.amazon.com/AWSEC2/latest/WindowsGuide/nvme-ebs-volumes.html#identify-nvme-ebs-device
+	ebsnvmeID := `C:\ProgramData\Amazon\Tools\ebsnvme-id.exe`
```

Comment:
minor nit - make it a constant?

Reply:
done

Reply:
I am sorry, I did not mean const defined inline here but on top of the file

Reply:
done x2